ui/src/audioplayer/Player.js

```diff
@@ -62,17 +70,28 @@ const AudioTitle = React.memo(({ audioInfo, isMobile }) => {
   return (
     <Link to={`/album/${audioInfo.albumId}/show`} className={className}>
       <span className={`${className} songTitle`}>
-        {audioInfo.name}
+        <span
+          onMouseEnter={() => setSongTitleHover(true)}
```

Comment:
I rather not do this with JavaScript. Can you find a solution with pure CSS? Ex: the context menu and heart are toggled with pure CSS